server/src/storage/commits.ts

```diff
@@ -101,13 +101,22 @@ const updateCommit = async (
         value: 1,
       });
       break;
-    case 'completed':
+    case 'completed': {
```

Comment:
redundant { ?

Reply:
@ifhuang I added the block because there is variable declaration which is block-scoped. Without the block, "the lexical declaration is visible in the entire switch block but it only gets initialized when it is assigned, which will only happen if the case where it is defined is reached." (https://eslint.org/docs/rules/no-case-declarations)